db/migrate/20240122064113_create_ad_campaigns.rb

```diff
@@ -0,0 +1,35 @@
+class CreateAdCampaigns < ActiveRecord::Migration[7.1]
+  def change
+    create_table :ad_campaigns, :id => false, primary_key: :ad_campaign_id do |t| 
+      t.string        :ad_campaign_id, null: false
+      t.string        :ad_campaign_name
+      t.string        :ad_campaign_objective
+      t.datetime      :ad_campaign_startdate
+      t.integer       :ad_campaign_lifetime_budget
+      t.string        :ad_campaign_budgeting_type
+      t.string        :ad_account_id, null: false
+      t.timestamps
+
+      t.index [:ad_campaign_id,:ad_account_id], unique:true
+    end
+
+    create_table :ad_campaigns_metrics do |t|
+      t.integer     :clicks
+      t.float       :ctr
+      t.integer     :link_clicks
+      t.float       :linl_clicks_ctr
```

Comment:
how do you plan to handle ratio metrics and in general metrics that can be derived from other metrics?